src/steamship/agents/mixins/transports/slack.py

```diff
@@ -5,20 +5,30 @@
 
 import requests
 from pydantic import BaseModel, Field
-
 from steamship import Block, Steamship
+from steamship import DocTag
 from steamship.agents.llms import OpenAI
 from steamship.agents.mixins.transports.transport import Transport
 from steamship.agents.schema import EmitFunc, Metadata
 from steamship.agents.service.agent_service import AgentService
 from steamship.agents.utils import with_llm
+from steamship.data import TagValueKey
+from steamship.data.block import get_tag_value_key
 from steamship.invocable import Config, InvocableResponse, get, post
 from steamship.utils.kv_store import KeyValueStore
 
 SLACK_API_BASE = "https://slack.com/api/"
 SETTINGS_KVSTORE_KEY = "slack-transport"
 
 
+def get_block_thread_ts(block: Block) -> Optional[str]:
+    return get_tag_value_key(block.tags, TagValueKey.STRING_VALUE, kind=DocTag.CHAT, name="slack-threadts")
```

Comment:
Since a `thread-id` is something in common to many chat clients, I think it might be nice to add a `THREAD_ID = 'thread-id'` constant to `ChatTags` here:

https://github.com/steamship-core/python-client/blob/4ecd7f15a7f2ca6c5390a5d01123b484d433f797/src/steamship/data/tags/tag_constants.py#L261

Then we can just record it as a transport-agnostic `thread-id` and the `slack.py` transport is "that which converts to our global schema"